src/xngin/apiserver/routers/experiments/test_experiments_common.py

```diff
@@ -140,6 +142,54 @@ def make_createexperimentrequest_json(
                     ],
                 }
             }
+        case ExperimentsType.CMAB_ONLINE:
+            context1_id = tables.context_id_factory() if with_ids else None
+            context2_id = tables.context_id_factory() if with_ids else None
+            return {
+                "design_spec": {
+                    **({"experiment_id": experiment_id} if experiment_id is not None else {}),
```

Comment:
(I can remove this and some other deprecated/removed usages in #47 if this pr merges first.)